.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Full Tests
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
To support all the extras on Windows and Python 3.11 we would need to drop `cdms2`.
xref.: https://github.com/conda-forge/cdms2-feedstock/issues/80#issuecomment-1309980939 and https://github.com/conda-forge/cdtime-feedstock/pull/52#issuecomment-1302342070

Reply:
Maybe the time is coming to remove the cdms2 interface in this module...

Reply:
But we can leave it like this for now and deal with that separately.